docs/docs/userguide/CONFIGURATION.md

```diff
@@ -157,3 +157,43 @@ db_allocated_storage = 100
 ```terraform
 db_iops = 1000
 ```
+
+### Bamboo License
+`bamboo_license` takes the license key of Bamboo product.
+
+To avoid storing license in a plain text file, we recommend storing it in an environment variable prefixed with `TF_VAR_`.
```

Comment:
```suggestion
To avoid storing license in a plain text file, we recommend storing it in an environment variable prefixed with [`TF_VAR_`](https://www.terraform.io/docs/cli/config/environment-variables.html#tf_var_name).
```